### ISSUE TITLE
Alerts IN STOCK for Not Available items on Amazon

### ISSUE THREAD

**Issue by Undeadhunter** [jef/nvidia-snatcher#440]
### Description

The application reports items to be in stock, even when not available for amazon-nl 

### Possible solution

Shouldn't alert when not available
Example: Alerts for https://www.amazon.nl/3080-GAMING-TRIO-10G-grafische/dp/B08HM4V2DH which isn't available, but doesn't have the buying option disabled.

**Comment by jef** [comment]
Need to add an `outOfStock` label. I will or someone will fix.

Thanks.

**Comment by xninjax** [comment]
I made an attempt on the weekend to find an `outOfStock` label for this.  However,  it does not appear that the page contents returned via the script ever contains this obvious oos message:

![image](https://user-images.githubusercontent.com/23090488/95126850-4dbe7580-0714-11eb-9100-a059d45089e5.png)

tried these oos labels:

```
outOfStock: {
			container: '#outOfStock > div > div.a-section.a-spacing-small.a-text-center > span',
			text: ['Tijdelijk niet op voorraad.']
		}
```

```
outOfStock: {
			container: 'div[id="outOfStock"] div div span',
			text: ['Tijdelijk niet op voorraad.']

		}
```
Also tried changing the user-agent (in case amazon was blocking the requests).

A debug dump of the entire page contents never contains the words: `Tijdelijk...` Yet to no avail was the expect output in the returned results... 😕 

Amazon using some clever js?  ...this one may take some extra digging... 🤷‍♂️

**Comment by jef** [comment]
Thanks for looking at this @xninjax. Bummer that it's not around. Looks like you've done enough digging. Probably not available to be parsed.

Going to close this out and then reference this ticket if it gets solved.

Thanks!
